Makefile

```diff
@@ -158,6 +158,16 @@ push-docker: build-docker
 $(MOCKS_DIR):
 	mkdir -p $(MOCKS_DIR)
 
+build-e2e-chain-init:
```

Comment:
@ramacarlucho Are these commands only used for testing upgrades? If this is the case, can we give these commands more meaningful names, that specify: 
* that they are e2e tests for upgrades
* which node to run, e.g. `pre-upgrade` and `post-upgrade`